TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -406,31 +406,37 @@ namespace Plugin {
 
     void TextToSpeechImplementation::onTTSStateChanged(bool state)
     {
+        TTSLOG_INFO("Notify onTTSStateChanged, state: %d", state);
```

Comment:
Please ensure the format we print the state & id values (i.e for state should we just print 0/1 or should it be true / false, likewise for id, should it be %d  or %u) are same as previous version.

Reply:
@varumugam123 
**For state :** I will change it into true/false

**For id:** https://github.com/rdkcentral/rdkservices/blob/sprint/23Q4/TextToSpeech/TextToSpeechImplementation.cpp#L320 I referred from here,  used %d as format specifier